org.sugarlabs.Musicblocks.json

```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "org.sugarlabs.Musicblocks",
```

Comment:
I highly recommend to rename to "...MusicBlocks".

Reply:
Note that, if we change it, these files are must be changed:
https://github.com/sugarlabs/musicblocks-launcher/tree/master/data